plugins/BEdita/Core/src/Model/Entity/AsyncJob.php

```diff
@@ -0,0 +1,83 @@
+<?php
+namespace BEdita\Core\Model\Entity;
+
+use Cake\I18n\Time;
+use Cake\ORM\Entity;
+
+/**
+ * Asynchronous Job Entity
+ *
+ * @property string $uuid
+ * @property string $service
+ * @property int $priority
+ * @property array $payload
+ * @property \Cake\I18n\Time $not_before
+ * @property \Cake\I18n\Time $not_after
+ * @property int $max_attempts
+ * @property \Cake\I18n\Time $locked_until
+ * @property \Cake\I18n\Time $created
+ * @property \Cake\I18n\Time $modified
+ * @property \Cake\I18n\Time $completed
+ * @property string $status
+ *
+ * @since 4.0.0
+ */
+class AsyncJob extends Entity
+{
+
+    /**
+     * {@inheritDoc}
+     */
+    protected $_accessible = [
+        'uuid' => true,
+        'service' => true,
+        'priority' => true,
+        'payload' => true,
+        'not_before' => true,
+        'not_after' => true,
+        'max_attempts' => true,
+    ];
+
+    /**
+     * {@inheritDoc}
+     */
+    protected $_virtual = [
+        'status',
+    ];
+
+    /**
+     * Magic getter for status.
+     *
+     * @return string
+     */
+    protected function _getStatus()
```

Comment:
Custom finder is required anyways: if a job's `status` is _pending_, it might have become invalid in the meanwhile, if it's _locked_, the lock might have expired, if it's _planned_, it might have become valid, …